.github/workflows/android.yml

```
@@ -57,11 +56,12 @@ jobs:
 
       - name: 💐 Install Qt
         if: ${{ steps.vars.outputs.INSTALL_QT }}
-        uses: jurplel/install-qt-action@v2
+        uses: jurplel/install-qt-action@v3
         with:
           version: 5.14.2
           modules: 'qtcharts'
-          target: ${{ steps.vars.outputs.QT_TARGET }}
+          target: ' android'
```

Is the extra space a typo?

```suggestion
          target: 'android'
```

---

That was on purpose but didn't help.